src/services/user/firstLogin.js

```diff
@@ -58,13 +58,6 @@ const firstLogin = async (data, params, app) => {
 		accountPromise = await app.service('accounts').patch(accountId, accountUpdate, params);
 	}
 
-	if (data.parent_email) {
```

Comment:
What about line 24? It will still create users there, right?

Reply:
Well this is a part of a lambda function code which call has been deleted. I will delete this lambda func as now it is never used.